src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Program.cs

```diff
@@ -11,6 +18,10 @@ namespace Altinn.Platform.Authorization
     /// </summary>
     public static class Program
     {
+        private static Logger logger = new LoggerConfiguration()
```

Comment:
Will this not conflict the logger configuration for serilog in line 44?

Reply:
My understanding is that the logger configured here is used for logging when program.cs is running. The one configured in line 44 is for the applications that are buildt using the host builder?

Reply:
renamed the one in line 44 to serilogLogger